_episodes/13.md

```diff
@@ -0,0 +1,449 @@
+will cover shortly. Xiang has a really great summary slide that easily shows
+the strengths of each system and why they work so well together.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/13/latency_flexibility.png"/><br/>
+</p>
+
+While Trino is not as fast as Pinot, it is able to handle a broader set of
+use cases like performing broad joins over open data formats in data lakes. 
+This is what motivated work on the Trino Pinot Connector. You can have the speed
+of Pinot, while having the flexibilty of Trino.
+
+Now that you understand the common use case for Pinot, it's important to know 
+the main goals of Pinot. 
+ 1. One primary goal is the keep response times of aggregation queries
+  predictable, regardless of how many requests Pinot handles. As it scales
+  you won't see a degradation of performance. This is achieved by Pinot's
+  custom indices and storage formats. 
+    <p align="center">
+       <img align="center" width="75%" height="100%" src="/assets/episode/13/data_value.jpeg"/><br/>
+    </p>
+
+ 2. Another goal of Pinot is to revive the value of data from a historical
+  context. Data will reach a particular point in its lifecycle where it
+  becomes less valuable as it ages. While all data is able to add some value no
+  matter what the age, there's a tradeoff of scanning multiple rows to glean
+  information from antiquated data. Pinot aims to remove this tradeoff as most 
+  questions around historical data are queried in aggregate and this can be
+  summarized and queried at a low cost.
+ 
+ 3. The final goal is to manage dimension explosion. One of the difficulties
+  with managing a system that caches all this historic data is handling
+  dimension explosion that occurs when you cache every possibly combination of
+  data. Above we showed a 3 dimensional cube, but Pinot can handle a much
+  larger number of dimensions. However, just because you can, doesn't mean you
+  should. Pinot has a lot of smarts around using the data, and some good
+  defaults to determine the maximum number of buckets per dimension. This helps
+  balance an exploding cache yet maintains fast results. 
+
+### Pinot Architecture
+
+We just covered Pinot theory and goals, let's take a quick look at their
+architecture.
+
+A [Pinot cluster](https://docs.pinot.apache.org/basics/components/cluster) 
+consists of a [controller](https://docs.pinot.apache.org/basics/components/controller), 
+[broker](https://docs.pinot.apache.org/basics/components/broker), 
+[server](https://docs.pinot.apache.org/basics/components/server), and
+optionally a [minion](https://docs.pinot.apache.org/basics/components/minion)
+to purge data.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/13/pinot_architecture.svg"/><br/>
+</p>
+
+## PR of the week: PR 2028 Add pinot connector
+
+In this week's pull request <https://github.com/trinodb/trino/pull/2028> that
```

Comment:
Our guest on the show today, Elon Azouley, is the author of this PR, so we can ask him all about it now.